lib/src/mock.dart

```diff
@@ -464,7 +464,7 @@ class RealCall {
           .map((key) =>
               "${_symbolToString(key)}: ${invocation.namedArguments[key]}")
           .join(", ");
-      args += ", {$namedArgs}";
+      args += (args.isEmpty) ? "{$namedArgs}" : ", {$namedArgs}";
```

Comment:
Nit can you separate this? I almost thought this was the same logic since the comma blends in. I think below would be easier to read, but understand if you want to leave the possibly slightly more performant version.

if (args.isNotEmpty) args += ", ";
args += "{$namedArgs}";

Reply:
Done!